doc/README.hi.md

```diff
@@ -0,0 +1,143 @@
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_02.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_02.png)
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_03.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_03.png)
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_04.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_04.png)
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_05.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_05.png)
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_06.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_06.png)
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_07.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_07.png)
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_08.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_08.png)
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_09.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_09.png)
+[<img src="../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_10.png" width=160>](../fastlane/metadata/android/en-US/images/phoneScreenshots/shot_10.png)
+[<img src="../fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_11.png" width=405>](../fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_11.png)
+[<img src="../fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_12.png" width=405>](../fastlane/metadata/android/en-US/images/tenInchScreenshots/shot_12.png)
+
+## विवरण
+
+NewPipe ना ही किसी Google फ्रेमवर्क लाइब्रेरी, और ना ही YouTube के API का इस्तेमाल करता है। सिर्फ आवश्यक जानकारी प्राप्त करने के लिए वेबसाइटों को पार्स किया जाता है तो इस ऐप का इस्तेमाल उन डिवाइसों पर भी किया जा सकता है जिनपर Google की सेवाएँ स्थापित नहीं हैं। और NewPipe जैसे कॉपीलेफ्ट किए गए मुक्त सॉफ़्टवेयर का इस्तेमाल करने के लिए आपको YouTube खाते की ज़रूरत नहीं।
+
+### सुविधाएँ
+
+* वीडियो खोजें
+* लॉग-इन करने की ज़रूरत नहीं
+* वीडियो के बारे में साधारण जानकारी पाएँ
+* YouTube के वीडियो देखें
+* YouTube के वीडियो सुनें
+* पॉपअप मोड (फ़्लोटिंग प्लेयर)
+* वीडियो देखने के लिए स्ट्रीम करने का प्लेयर चुनें
+* वीडियो डाउनलोड करें
+* सिर्फ ऑडियो डाउनलोड करें
+* Kodi में वीडियो को खोलें
+* अगले/संबंधित वीडियो देखें
+* YouTube को किसी विशिष्ट भाषा में खोजें
+* उम्र से जुड़ी पाबंदी वाली सामग्री देखें/रोकें
+* चैनलों के बारे में साधारण जानकारी पाएँ
+* चैनल खोजें
+* किसी विशिष्ट चैनल से वीडियो देखें
+* Orbot/Tor का समर्थन (सीधे नहीं)
+* 1080p/2K/4K का समर्थन
+* देखे गए वीडियो देखें
+* चैनलों पर सदस्यता लें

```

Comment:
Sure about that? In this and [the next convo](https://github.com/TeamNewPipe/NewPipe/pull/8355#discussion_r867331497) I've followed YouTube's official translations, if you can confirm usage of popular terms I can change it.

Reply:
Okay. Got it.

Still, YT's subscription is account-based. Ours is not. Newpipe just adds that channel to the main page tab and adds its new videos to the What's New feed. It's like getting RSS updates of a news website, instead of making an account on it and getting notifications.

So the popular terms don't apply here.